sdk/src/assertion.rs

```diff
@@ -202,10 +202,12 @@ impl fmt::Debug for AssertionData {
     }
 }
 
-/// Standard Assertion data types.  Each assertion type will
-/// contain its AssertionData.  For the User Assertion type we
-/// allow a String to set the label. The AssertionData contains
-/// the data payload for the assertion and the version number for its schema (if supported).
+/// Internal Assertion structure
```

Comment:
As we discussed on Slack, not sure what to do with this since `AssertionData` is not exposed from the crate.

Reply:
I changed the references to AssertionData to not be in public comments, since it is not exposed

Reply:
I just removed the public comments about AssertionData so they don't show up in the docs - also corrected this so that the first line is only one line and not a big paragraph.